danielMunteanu/src/main/java/danielCollectionTask/AbstractUser.java

```diff
@@ -0,0 +1,36 @@
+package danielCollectionTask;
+
+import java.util.Map;
+import java.util.UUID;
+
+public abstract class AbstractUser {
+    final String username;
```

Comment:
can we make these properties private?
Otherwise we we are creating the getters and setters?

Reply:
fixed